e2e/scripts/verdaccio.yaml

```diff
@@ -19,7 +19,7 @@ packages:
   '@tensorflow/**':
     access: $all
     publish: $all
-    # Don't set proxy to avoid accidentally publishing to npm.
```

Comment:
This should not be possible without an OTP, so it's okay to remove this.

Reply:
I think this statement is incorrect, the `proxy` property does not allow publishing on npmjs, in fact only works in one way, from npmjs to your local cache. If you set `proxy: npmjs` what is gonna happens is when you publish a package that match `@tensorflow/**` will ask first npmjs if already exist there or not, if you are publishing a snapshot with a random version probably will goes through otherwise you will have. 409 publish conflict.

Reply:
Thanks for the clarification. I was wondering if this comment was correct, and I couldn't find any description of this publishing behavior in the docs.

Reply:
Thanks I'll try to highlight it better 👍🏼